examples/mksomersaultmodifiedpdx.py

```diff
@@ -0,0 +1,230 @@
+#! /usr/bin/python3
+#
+# SPDX-License-Identifier: MIT
+#
+# This script demonstrates how an existing ODX database can be
+# modified programatically. Note that this is pretty hacky...
+import argparse
+from copy import deepcopy
+from typing import List, TypeVar
+
+import odxtools
+import odxtools.uds as uds
+from examples import somersaultecu
+from odxtools.diaglayer import DiagLayer
+from odxtools.diagservice import DiagService
+from odxtools.nameditemlist import NamedItemList
+from odxtools.odxlink import OdxLinkId, OdxLinkRef
+from odxtools.parameters.codedconstparameter import CodedConstParameter
+from odxtools.parameters.valueparameter import ValueParameter
+from odxtools.request import Request
+from odxtools.response import Response, ResponseType
```

Comment:
Looking at those imports, it reminds that we should expose those classes in the top level module, otherwise it gets hard to figure out where each class comes from

Reply:
I'm not opposed, but I don't see how this can be done without subscribing to a maintenance nightmare: As far as I'm aware, the only way to do this is to explicitly import  each and every class in the toplevel `__init__.py`?

Reply:
You are already doing that in `examples/somersaultecu.py`, you will just be moving imports from one file to another.

Reply:
in `__init__.py` there would be the implicit promise that the imports are complete whilst files containing code only import what they actually use (in fact, this is even enforced by `ruff`...)

Reply:
Maybe create issues ticket? so it does not get lost in this PR review.

Reply:
good point. done: https://github.com/mercedes-benz/odxtools/issues/287